js/build.js

```diff
@@ -178,6 +178,7 @@ Fliplet().then(function() {
                         action: 'authenticate_fail'
                       });
                       vmData.codeError = true;
+                      vmData.codeErrorMessage = Fliplet.parseError(error);
```

Comment:
does this variable need to be initialised as empty on the vue instance?

Reply:
@squallstar Well caught. Updated.